index.html

```diff
@@ -148,5 +147,22 @@ <h2>About Myself</h2>
             </div>
         </section>
     </main>
+    <footer class="contact">
+        <section class="fieldset">
+            <h1>Contact me</h1>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that
+                needs
+                coding.I’d love to help with it!</p>
+            <form action="https://formspree.io/f/xpzeyzyd" method="post">
+                <label for="fname"><input type="text" name="userName" id="fname" maxlength="30" placeholder="Full Name"
+                        required></label>
+                <label for="email"><input type="email" id="email" name="userEmail" placeholder="Email" required></label>
+                <label for="messages"><textarea name="message" maxlength="500" id="messages"
+                        placeholder="Write your message here" cols="30" rows="10" required></textarea></label>
+                <button type="submit">Get in touch</button>
```

Comment:
- Also, the button is missing the relevant colors, and background color when hovered, active, enabled, and disabled. kindly update this to improve the user experience of your website.

![image](https://github.com/BenMKT/myPortfolio/assets/59397159/330177b3-6154-4eaf-b258-5af0b6344676)

Reply:
- Added background color on hover and when active states.